### PR TITLE
Create components

### PR DESCRIPTION
**Pull request by areisle**
client side of wizzards so far

### CODE REVIEW

File: src/Context/fixtures.ts [hunk 1]
```diff
@@ -0,0 +1,179 @@
+// fixed game states for helping in development components
```
Comment (creisle) [inline src/Context/fixtures.ts]: can we make this comment a bit more explicit? like "TESTING ONLY" ? or similar

File: src/Main.tsx [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+    const [betDialogOpen, setBetOpen] = useState(false);
+
+    const isSetup = stage === 'awaiting-players';
+
+    const handleCloseScoreBoard = () => {
+        setBoardOpen(false);
+    }
+
+    /**
+     * @todo handle bet value
```
Comment (creisle) [inline src/Main.tsx]: leftover?

File: src/components/GameBoard/index.tsx [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+function Fillers(props: { players: number, show: boolean; }) {
+    const { players, show } = props;
+    if (!show) {
+        return null;
+    }
+
+    const shells = new Array(MAX_NUMBER_OF_PLAYERS - players).fill(null).map((_, index) => (
+        <PlayerAvatar
+            key={index + players}
+            player={index + players + 1}
```
Comment (creisle) [inline src/components/GameBoard/index.tsx:22]: this is the playerIndex?

File: src/components/Header/index.tsx [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+    trick: GameState['trickNumber'];
+    trump: GameState['trumpSuit'];
+}
+
+function Header(props: HeaderProps) {
+    const {
+        onScoreBoardOpen,
+        round,
+        stage,
+        trick,
```
Comment (creisle) [inline src/components/Header/index.tsx]: sometimes you call this trick and other times trickNumber. Might be less confusing if they were consistent?

File: src/components/PlayingCard/index.tsx [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+    selected?: boolean;
+    onClick?: HTMLProps<HTMLDivElement>['onClick'];
+    children?: HTMLProps<HTMLDivElement>['children'];
+}
+
+function Marker(props: Partial<Card>) {
+    const { suit, number } = props;
+    const letter = suit?.charAt(0).toUpperCase();
+    return (
+        <div className={`marker marker--${suit}`}>
```
Comment (creisle) [inline src/components/PlayingCard/index.tsx:17]: should we add a todo comment here for replacing these with icons?

File: src/components/Rules/index.tsx [hunk 6]
```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import { Typography, Table, TableBody, TableRow, TableCell, TableHead, Card, CardContent } from '@material-ui/core';
+
+interface RulesProps {
```
Comment (creisle) [inline src/components/Rules/index.tsx]: why is this interface empty?

**Review by creisle — APPROVED** [review]

File: src/components/GameBoard/index.tsx (hunk 3 above)
Comment (areisle) [reply src/components/GameBoard/index.tsx:22]: it's the player index + 1; this is for the styling and positioning of the avatar component

File: src/components/Header/index.tsx (hunk 4 above)
Comment (areisle) [reply src/components/Header/index.tsx]: I could change them all to trickNumber? I have been using trick > trickNumber for brevity, but only when there isn't other trick things to make it confusing

File: src/components/PlayingCard/index.tsx (hunk 5 above)
Comment (areisle) [reply src/components/PlayingCard/index.tsx:17]: no but we can make a ticket for it though; I don't plan on looking at that until after user testing

File: src/components/Rules/index.tsx (hunk 6 above)
Comment (areisle) [reply src/components/Rules/index.tsx]: thought I would need props when I was scaffolding; I'll remove it

File: src/Context/fixtures.ts (hunk 1 above)
Comment (areisle) [reply src/Context/fixtures.ts]: f7595c2a0c7354d091246e98611889f42fdc864b

File: src/Main.tsx (hunk 2 above)
Comment (areisle) [reply src/Main.tsx]: f7595c2a0c7354d091246e98611889f42fdc864b

File: src/components/Rules/index.tsx (hunk 6 above)
Comment (areisle) [reply src/components/Rules/index.tsx]: f7595c2a0c7354d091246e98611889f42fdc864b

File: src/components/PlayingCard/index.tsx (hunk 5 above)
Comment (areisle) [reply src/components/PlayingCard/index.tsx:17]: https://github.com/creisle/quarantined_wizzards/issues/3

File: src/components/Header/index.tsx (hunk 4 above)
Comment (areisle) [reply src/components/Header/index.tsx]: addressed https://github.com/creisle/quarantined_wizzards/pull/1/commits/110115d5b48a83680bf13fdd560c486cc1988257

**Pull request merged**